install-deps.sh

```diff
@@ -342,7 +342,7 @@ else
 	$SUDO env DEBIAN_FRONTEND=noninteractive mk-build-deps --install --remove --tool="apt-get -y --no-install-recommends $backports" $control || exit 1
 	$SUDO env DEBIAN_FRONTEND=noninteractive apt-get -y remove ceph-build-deps
 	if [ "$control" != "debian/control" ] ; then rm $control; fi
-	$SUDO apt-get install -y libxmlsec1 libxmlsec1-nss libxmlsec1-openssl libxmlsec1-dev
+	$SUDO apt-get install -y libxmlsec1 libxmlsec1-nss libxmlsec1-openssl libxmlsec1-dev libcap-ng-dev
```

Comment:
no need to install `libcap-ng-dev` separately. `debian/control` should already take care of this.

these dependencies are only for building the [python-saml](https://github.com/onelogin/python-saml) python extension. the SAML feature is not enabled by default.